content/en/blog/2024-01-26-monitor-real-time-ml-apps-with-prefect.md

```diff
@@ -31,7 +31,7 @@ Prefect offers a lot of neat features but what’s even more awesome is that get
 3. Run the following command in a terminal window: `prefect server start` (note that your virtual environment must be activated before running this command).
 4. Run your application and view the live status of the application in the Prefect dashboard.
 
-The example I will demonstrate is a real-time sentiment analysis application that uses [Ensign](https://rotational.io/ensign/) as the streaming data platform, [River](https://riverml.xyz) for the real-time machine learning model, and [Prefect]((https://www.prefect.io)) as the orchestration platform. We'll use this sample dataset of [Yelp reviews](https://github.com/rotationalio/prefect-example/tree/main/data) as a stand-in for the data streaming into our to-be-deployed model.
```

Comment:
This fixes this error:

```
"/Users/rebeccabilbro/Desktop/eudicot/rotationalio/rotationalio.github.io/content/en/blog/2024-01-26-monitor-real-time-ml-apps-with-prefect.md:1:1": execute of template failed: template: _default/_markup/render-link.html:1:14: executing "_default/_markup/render-link.html" at <urls.Parse>: error calling Parse: parse "(https://www.prefect.io/)": first path segment in URL cannot contain colon
```

I believe before hugo 0.125, this error was ignored, and the markdown was rendered the link as `https://rotational.io/(https://www.prefect.io)`. Hugo must have fixed that bug.

Reply:
CC @rebeccabilbro @daniellemaxwell @pdamodaran